### PR TITLE
Add v4.2.0 Release

### PR DESCRIPTION
**Pull request by jcudit**
@frbayart @patrickmarabeas @majormoses @Polygens thanks for your contributions 🙇🏾

### CODE REVIEW

**Comment by shoekstra** [comment]
Hey @jcudit - would love to get the bug fix in #644 in on this release if possible 🙂

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+## 4.1.1 (Unreleased)
```
Comment (majormoses) [inline CHANGELOG.md:1]: If there are enhancements than this should be a `4.2.0` release to follow semver.

**Review by majormoses — APPROVED** [review]
LGTM after updating the version to reflect the impact of the changes.

File: CHANGELOG.md (hunk 1 above)
Comment (jcudit) [reply CHANGELOG.md:1]:
```suggestion
## 4.2.0 (Unreleased)
```

**Pull request merged**